src/OpenDebugAD7/AD7DebugSession.cs

```diff
@@ -765,6 +776,8 @@ protected override void HandleLaunchRequestAsync(IRequestResponder<LaunchArgumen
                     eb.ThrowHR(hr);
                 }
 
+                Protocol.SendEvent(new CapabilitiesEvent() { Capabilities = new InitializeResponse() { SupportsStepBack = ((IDebugReversibleEngineProgram160)m_engine).CanReverse() != 0 } });
```

Comment:
```suggestion
                Protocol.SendEvent(new CapabilitiesEvent() { Capabilities = new InitializeResponse() { SupportsStepBack = ((IDebugReversibleEngineProgram160)m_engine).CanReverse() == HRConstants.S_OK } });
```

Reply:
@Trass3r Have you checked to see if this changes the VS Code UI to show the reverse debug buttons?

Reply:
Oh yeah it works fine both ways 🎉